tests/testthat/test-assertions.R

```diff
@@ -0,0 +1,112 @@
+library(admiraltest)
+
+test_that("an error is thrown if a required variable is missing", {
+  data(dm)
+
+  expect_error(
+    assert_has_variables(dm, "TRT01P"),
+    "Required variable `TRT01P` is missing."
+  )
+})
+
+test_that("no error is thrown if a required variable exists", {
```

Comment:
same comment as above, add function name to `test_that` title, and maybe Test number?